gitops/azure-devops/build.sh

```diff
@@ -168,7 +168,7 @@ function git_commit() {
     rm -rf ./*/
     git rm -rf ./*/
     echo "COPY YAML FILES TO REPO DIRECTORY..."
-    cp -r "$HOME/generated/*" .
+    cp -r $HOME/generated/* .
```

Comment:
```suggestion
    cp -r "$HOME/generated/"* .
```

Reply:
Sorry about that, that was a typo on my part. The quotes aren't the issue; you should keep the qutoes so that spaces in $HOME don't split the command. It's the shell expansion that gets squashed inside string, the star should go outside the quotes to complete the expansion. 

See https://stackoverflow.com/questions/46346638/copy-files-with-double-quotes-and-matching-pattern

Reply:
No worries, thanks @sayar for the suggestion!